.github/workflows/meetings.yml

```diff
@@ -0,0 +1,24 @@
+name: Schedule Meetings
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '0 0 * * 1'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Technical Committee
+        uses: 'pkgjs/meet@v0'
+        with:
+          issueTitle: '<%= date.toFormat("yyyy-MM-dd") %> Express TC Meeting'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          orgs: expressjs,pillarjs,jshttp
+          agendaLabel: 'top priority'
```

Comment:
I'd expect this to be `agenda` or an array that allows `['top-priority', 'agenda']`

Adding an `agenda` label is the convention I'd expect. Easy to slap an agenda label on something to know it will show up in the meeting. Not everything to be discussed is always going to be "top priority"

Reply:
Yeah I only used this to start because it was preexisting. We can go through all of them and add a better label for future.